### ISSUE TITLE
Bug: Should the Slack link be the invitation link on mega dropdown menu?

### ISSUE THREAD

**Issue by HuangJunye** [Qiskit/web-components#158]
### Steps to reproduce the problem

1. Go to https://qiskit.org/
2. Click "Documentation" dropdown menu
3. Check the link for "Slack"

### What is the current behavior?

The current link is https://qiskit.slack.com/

### What is the expected behavior?

We have been using https://ibm.co/joinqiskitslack (which is a bit.ly link for the Slack invitation) for the link to Slack so that new users can also join. The current link https://qiskit.slack.com/ only works for users that already joined Slack. 

But I know there is also discussions about reducing the number of users to our Slack so this link https://qiskit.slack.com/ could be a better link. 

cc: @lerongil @1ucian0

**Comment by paulsearle** [comment]
We should use the bit.ly link so new members can join the Slack, good catch.

**Comment by lerongil** [comment]
Yes, agreed. https://ibm.co/joinqiskitslack should be the only slack url we point to.
